ldes-server-admin/pom.xml

```diff
@@ -33,6 +33,32 @@
             <artifactId>swagger-annotations-jakarta</artifactId>
         </dependency>
 
+        <!-- LOGGING -->
```

Comment:
Just a bit of nitpicking, let's call this "Monitoring and Logging" perhaps :) since i see prometheus is also part of this